.travis.yml

```diff
@@ -0,0 +1,11 @@
+---
+language: node_js
+
+node_js:
+ - "stable"
```

Comment:
Super-weird that we can use `4.3.2`, especially since that's the LTS version? 

Weird. https://docs.travis-ci.com/user/languages/javascript-with-nodejs#Provided-Node.js-Versions